includes/class-cache.php

```
@@ -376,10 +405,241 @@ public function combine_css() {
+					}
+				}
+
+				if ( ! isset( $style_data->args ) || 'all' !== $style_data->args ) {
+					continue;
+				}
+
+				$handles[] = $handle;
+			}
+
+			return $handles;
+		}
```

PHPMD (2.15.0)

[warning] The method get_combined_handles() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)